Makefile

```diff
@@ -490,10 +489,10 @@ ngt/install: /usr/local/include/NGT/Capi.h
 	cd $(TEMP_DIR)/NGT-$(NGT_VERSION) && \
 		cmake -DCMAKE_C_FLAGS="$(CFLAGS)" -DCMAKE_CXX_FLAGS="$(CXXFLAGS)" .
 	make -j -C $(TEMP_DIR)/NGT-$(NGT_VERSION)
-	make install -C $(TEMP_DIR)/NGT-$(NGT_VERSION)
+	sudo make install -C $(TEMP_DIR)/NGT-$(NGT_VERSION)
```

Comment:
Is it OK?
It seems it requires a pass during running `make ngt/install`.

Reply:
sorry for late reply.
the `make ngt/install` is executed in `make init` command, so it becomes more difficult to handle.

I suggest to ask user to execute `sudo make init` instead of running `sudo` here, what do you think?